src/main/resources/templates/db_scripts/schema.json

```diff
@@ -186,13 +186,23 @@
           "fieldName" : "name",
           "tOps" : "ADD"
         }
+      ],
+      "uniqueIndex": [
+        {
+          "fieldName" : "name"
+        }
       ]
     },
     {
       "tableName": "settings",
       "fromModuleVersion": "mod-organizations-storage-4.7.0",
       "withOptimisticLocking": "failOnConflict",
-      "withMetadata": true
+      "withMetadata": true,
+      "uniqueIndex": [
+        {
+          "fieldName" : "name"
```

Comment:
There is no field 'name' in settings table, there is field 'key' instead:
https://github.com/folio-org/acq-models/blob/master/mod-orgs/schemas/setting.json#L14

Reply:
thanks you, updated